raster-foundry/raster-foundry
# Fix project share urls using the api url instead of frontend url

## Overview
Fix project share urls using the api url instead of frontend url
Make the (i) icons display the url / token that you will copy by pressing the button


### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~

## Testing Instructions

 * Verify that if you set the BUILDCONFIG.API_HOST parameter to a url other than where the frontend is hosted, it will still use the correct url (eg `https://some.hosted.api` shouldn't result in `http://https://some.hosted.api/...`)
* Verify that the (i) icons actually do something for now

Closes https://github.com/azavea/raster-foundry-platform/issues/172
